lib/rubocop/cop/rspec/describe_class.rb

```diff
@@ -44,12 +44,22 @@ class DescribeClass < Cop
 
         def_node_matcher :shared_group?, SharedGroups::ALL.block_pattern
 
-        def on_top_level_describe(node, args)
+        def on_top_level_describe(node, (first_arg, _))
```

Comment:
What would be the best name for the first argument? Maybe `described_class`?

Reply:
I've took this approach from `lib/rubocop/cop/rspec/describe_method.rb` (`second_arg`), but I think `described_value` is better, yes.